conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/processing/BakersProcessor.scala

```diff
@@ -231,6 +205,14 @@ class BakersProcessor(
       case InvalidPositiveDecimal(_) => None
     }
 
+    /** Computes updated rolls from the stake balance being updated */
+    def rollsFromStakes(balance: PositiveBigNumber): Int = balance match {
```

Comment:
How come the same logic is replicated in two places, i.e. here as well as `TezosTypes`? Might caught confusion and bugs later.

Reply:
I've spotted this additional change late evening, and wanted something quick up for you all to review.

I'll try to clean up now

Reply:
Easier than I thought